### DISCUSSION TITLE
Using the windows executable how to configure?

### DISCUSSION THREAD

**Discussion by elcosmelco** [jesec/flood#298] [Q&A]
Using the packaged windows executable, how do I configure Flood? To be exact, I am able to view the interface locally via the default url ( http://localhost:3000 ), but not via any other device on the network resulting in a `ERR_CONNECTION_REFUSED`. I am assuming I have to bind to 0.0.0.0 in the config where it is currently bound to 127.0.0.1?

**Comment by jesec** [comment]
Open a PowerShell or `cmd.exe` prompt, and then follow the instructions of "Configuration" section of README.

![image](https://user-images.githubusercontent.com/11585141/116788562-e6677900-aadc-11eb-818d-7f4b39d60e24.png)

**Reply by elcosmelco** [reply]
excellent I will give this a go! thanx for the quick response!
